app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/DataSetTableScreen.kt

```diff
@@ -0,0 +1,90 @@
+package org.dhis2.usescases.datasets.dataSetTable.dataSetSection
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.BottomSheetScaffold
+import androidx.compose.material.BottomSheetValue
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.rememberBottomSheetScaffoldState
+import androidx.compose.material.rememberBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.unit.dp
+import com.google.android.material.composethemeadapter.MdcTheme
+import kotlinx.coroutines.launch
+import org.dhis2.composetable.model.TableCell
+import org.dhis2.composetable.model.TableModel
+import org.dhis2.composetable.model.TextInputModel
+import org.dhis2.composetable.ui.TableColors
+import org.dhis2.composetable.ui.TableList
+import org.dhis2.composetable.ui.TextInput
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun DataSetTableScreen(
```

Comment:
If we are going to use it for something yes. 
Marcos is not checking them, and we always launch the app to test the component. So it may be open for discussion